RELEASE-NOTES.md

```diff
@@ -1,5 +1,74 @@
 # Prefect Release Notes
 
+## Release 2.10.6
+
+### Deploy many flows at once with projects
```

Comment:
```suggestion
### Deploy many flows at once with Projects
```

Reply:
We do not capitalize Prefect concepts afaik

Reply:
Makes sense-- it looks like we don't consistently capitalize our highlight headings either